pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>100</version>
+        <version>101</version>
```

Comment:
> Build server for Java 11

Maybe add to cmt msg some hint that requiring Java 11+ for compilation (of every module) is conscious choice.

Reply:
Good idea, updated